src/pages/steps/AdditionalInformationStep/DescriptionStep.tsx

```diff
@@ -138,7 +138,9 @@ const DescriptionStep = ({
     );
 
     changeDescriptionMutation.mutate({
-      description: draftToHtml(convertToRaw(editorState.getCurrentContent())),
+      description: plainTextDescription.length
+        ? draftToHtml(convertToRaw(editorState.getCurrentContent()))
+        : '',
```

Comment:
```suggestion
      description: plainTextDescription.length > 0
        ? draftToHtml(convertToRaw(editorState.getCurrentContent()))
        : '',
```

To be a bit more explicit